File: back/model/booking.go
package model

type NewBooking struct {
	UserID        int64
	RoomID        int64
	ReservedNum   int64
	BeginDateTime string
	EndDateTime   string
}

type Booking struct {
	ID            int64
	UserID        int64
	RoomID        int64
	ReservedNum   int64
	BeginDateTime string
	EndDateTime   string
}

File: front/model/booking.py
import dataclasses


@dataclasses.dataclass
class NewBooking:
    user_id: int
    room_id: int
    reserved_num: int
    begin_date_time: str
    end_date_time: str


@dataclasses.dataclass
class Booking(NewBooking):
    id: int
